src/index.html

```diff
@@ -6,9 +6,57 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="head__wrap">
+        <div class="logo">
+          <a href="index.html">
+            <img
+              alt="logo"
+              src="images/logo.png"
+              class="logo__img"
+            >
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item ">
+              <a href="#" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a href="#"
+                 class="nav__link"
+                 data-qa="hover">
```

Comment:
Check the code style 
```suggestion
              <a 
                href="#"
                class="nav__link"
                data-qa="hover"
              >
```